### PR TITLE
[Test-Only] Test CI with new azdev to build against requirements file

### PR DESCRIPTION
**Pull request by fengzhou-msft**
**Description**  

Use this PR to trigger CI tests with changes in #13353 and https://github.com/Azure/azure-cli-dev-tools/pull/194

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Test CI with new azdev

File: .azure-pipelines/templates/automation_test.yml [hunk 1]
```diff
@@ -14,10 +14,10 @@ steps:
       chmod +x env/bin/activate
       source env/bin/activate
 
-      pip install -q azdev
+      pip install git+https://github.com/fengzhou-msft/azure-cli-dev-tools.git@dependency_choice -q
       azdev --version
 
-      azdev setup -c ./
+      azdev setup -c ./ -d requirements
```
Comment (fengzhou-msft) [inline .azure-pipelines/templates/automation_test.yml:20]:
```suggestion
      azdev setup -c ./ -d requirements.txt
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -149,8 +149,8 @@ jobs:
       pip install virtualenv
       python -m virtualenv venv/
       . ./venv/bin/activate
-      pip install azdev
-      azdev setup -c $(pwd)
+      pip install git+https://github.com/fengzhou-msft/azure-cli-dev-tools.git@dependency_choice
+      azdev setup -c $(pwd) -d requirements
```
Comment (fengzhou-msft) [inline azure-pipelines.yml:153]:
```suggestion
      azdev setup -c $(pwd) -d requirements.txt
```

File: scripts/ci/install_azdev.sh [hunk 3]
```diff
@@ -8,5 +8,5 @@ pip install virtualenv
 python -m virtualenv env
 . env/bin/activate
 pip install -U pip setuptools wheel -q
-pip install azdev==0.1.4 -q
-azdev setup -c
+pip install git+https://github.com/fengzhou-msft/azure-cli-dev-tools.git@dependency_choice -q
+azdev setup -c -d requirements
```
Comment (fengzhou-msft) [inline scripts/ci/install_azdev.sh:12]:
```suggestion
azdev setup -c -d requirements.txt
```

**Pull request closed without merging**